pyproject.toml

```diff
@@ -188,7 +191,6 @@ filterwarnings = [
 ]
 
 [tool.coverage.run]
-branch = true
```

Comment:
Why is it removed? It does seem to be useful to [have branch coverage](https://coverage.readthedocs.io/en/latest/branch.html#how-to-measure-branch-coverage)

Reply:
I tried several times, the ci failed with `branch=true`.
As we are using [pytest-cov](https://pytest-cov.readthedocs.io/en/latest/config.html), there is no need to add `[tool.coverage.run]` section in pyproject.toml
This PR https://github.com/tortoise/tortoise-orm/pull/1908 show that overall coverage does not reduce without `tool.coverage.run` section.

Reply:
@waketzheng, https://github.com/tortoise/tortoise-orm/pull/1908 actually shows that the test coverage improved because... execution branches aren't covered anymore. I don't think we need to remove this option.

Have you figured out why ci started to fail? is it dependency update or the pyproject.toml change that caused it?

Reply:
@waketzheng I applied your changes in [this PR ](https://github.com/tortoise/tortoise-orm/pull/1910/files) but I replaced `[tool.coverage.run]` with `--cov-branch` to `pytest_opts`. This solved the issue and left the coverage unchanged. I didn't understand yet what is the root cause of the issue but it looks like `[tool.coverage.run]` is not being used.

Reply:
@henadzit I think we can merge this PR, and start another one about `--cov-append/coverage.run`.

Reply:
@waketzheng I agree but let's add `--cov-branch` to pytest arguments so we don't change how we are calculating the code coverage. This PR shows an increase of code coverage by 2.2% which is not correct. I think with this change we should be able to merge it.

Reply:
@waketzheng please let me know if you don't have time updating this PR, I think we can merge [this one](https://github.com/tortoise/tortoise-orm/pull/1910/files) instead, it is based on yours and has `--cov-branch` added.